marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/step.sjs

```diff
@@ -77,20 +77,20 @@ class Step {
   }
 
   getStepsByType(type = 'custom') {
-    let query = [cts.collectionQuery('http://marklogic.com/data-hub/step-definition'), cts.jsonPropertyValueQuery('type', type, 'case-insensitive')];
+    let query = [cts.collectionQuery('http://marklogic.com/data-hub/step-definition'), cts.jsonPropertyValueQuery('type', type, ["unstemmed","case-insensitive"])];
```

Comment:
Any idea why some of these artifact queries are case-insensitive and others are not? This makes it seem that DHF is not sure if case sensitivity matters or not when finding a step. 

I am guessing that all artifact queries should be case-insensitive on "name" because the name is used in the filename. And it also doesn't seem beneficial to the user to allow for separate steps of "myIngester" and "myingester".

Reply:
I am not sure why, also this module step.sjs deals mostly with crud operation of step definition and not steps.